deployments/terraform/modules/sqs/variables.tf

```diff
@@ -64,3 +64,15 @@ variable "alarm_dead_evaluation_periods" {
 variable "alarm_dead_datapoints_to_alarm" {
   default = 1
 }
+
+variable "alarm_main_message_age_create" {
+  default = 1
+}
+
+variable "alarm_dead_message_age_create" {
```

Comment:
this one is obsolete?

Reply:
Why? by default for queue var alarm_message_age_create is 1, so if we dont overwrite it with 0 in sqs part it will be created, when actually should not.

Reply:
okay, i guess i would have set it to `0` here: https://github.com/applike/gosoline/blob/1e0f13e98e9b0e082e995b3813618251f42159f9/deployments/terraform/modules/sqs/main.tf#L47  

but we can also keep your approach